serialization.md

```diff
@@ -1,7 +1,7 @@
 # OCI Image Serialization
 
 An *Image* is an ordered collection of root filesystem changes and the corresponding execution parameters for use within a container runtime.
-This specification outlines the format of these filesystem changes and corresponding parameters and describes how to create and use them for use with a container runtime and execution tool.
+This specification outlines the JSON format describing images for use with a container runtime and execution tool and its relationship to filesystem change sets, described in [Layers](layer.md).
 
 ## Terminology
```

Comment:
On Fri, Aug 26, 2016 at 03:22:13PM -0700, Stephen Day wrote:
> Either way, this is massively outside the scope of this PR.

I have no problem punting to a follow-up PR.  No need to slow this one
down with the terminology move being contentious.